backup-and-restore-using-dumpling-and-lightning.md

```diff
@@ -0,0 +1,139 @@
+---
+title: 使用 Dumpling/TiDB Lightning 备份与恢复
+summary: 了解如何使用 Dumpling 和 TiDB Lightning 备份与恢复集群数据。
+---
+
+# 使用 Dumpling/TiDB Lightning 备份与恢复
+
+本文档介绍如何使用 Dumpling 和 TiDB Lightning 进行全量备份与恢复。
+
+在备份恢复过程中，我们会使用 [Dumpling](/dumpling-overview.md) 从 TiDB 导出数据进行备份，然后使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 导入数据至 TiDB 实现恢复。
+
+## 前提条件
+
+- [安装 Dumpling 和 TiDB Lightning。](/migration-tools.md)
```

Comment:
Please provide the more friendly anchor link.

Reply:
Made modification in https://github.com/pingcap/docs-cn/pull/10286/files#diff-5f4635ca14610f49f1a1c3d5c990ea927ae0bf6e26c6dd6b4f545021c5a0b7c6R14-R15. PTAL.